app/Domain/Auth/Controllers/Callback.php

```diff
@@ -21,7 +22,13 @@ public function __invoke(Request $request): RedirectResponse
             $user = $auth0->getUser();
 
             if ($user !== null) {
+                // httpOnly = false
+                Cookie::queue('auth.token.idToken', $auth0->getIdToken(), 3600 * 60 * 24, null, null, null, false);
+
                 Auth::login(UserModel::fromSession($user));
+            } else {
+                // Delete the cookie
+                Cookie::queue('auth.token.idToken', '', 0);
```

Comment:
```suggestion
                Cookie::expire('auth.token.idToken');
```

I think you can use this method